in_toto/gpg/rsa.py

```diff
@@ -0,0 +1,83 @@
+"""
+<Module Name>
+  gpg/rsa.py
+
+<Author>
+  Santiago Torres-Arias <santiago@nyu.edu>
+
+<Started>
+  Nov 15, 2017
+
+<Copyright>
+  See LICENSE for licensing information.
+
+<Purpose>
+  RSA-specific handling routines for signature verification and key parsing
+"""
+import binascii
+
+import cryptography.hazmat.primitives.hashes as hashing
+import cryptography.hazmat.primitives.asymmetric.rsa as rsa
+import cryptography.hazmat.backends as backends
+import cryptography.hazmat.primitives.asymmetric.padding as padding
+import cryptography.hazmat.primitives.asymmetric.utils as utils
+import cryptography.exceptions
+
+from in_toto.gpg.util import get_mpi_length, hash_object
+import in_toto.gpg
+
+def create_pubkey(pubkey_info):
+  e = int(pubkey_info['keyval']['public']['e'], 16)
+  n = int(pubkey_info['keyval']['public']['n'], 16)
+  pubkey = rsa.RSAPublicNumbers(e, n).public_key(backends.default_backend())
+
+  return pubkey
+
+def get_pubkey_params(data):
+  ptr = 0
+
+  modulus_length = get_mpi_length(data[ptr: ptr + 2])
+  ptr += 2
+  modulus = data[ptr:ptr + modulus_length]
+  if len(modulus) != modulus_length: # pragma: no cover
+    raise in_toto.gpg.PacketParsingError("This modulus MPI was truncated!")
+  ptr += modulus_length
+
+  exponent_e_length = get_mpi_length(data[ptr: ptr + 2])
+  ptr += 2
+  exponent_e = data[ptr:ptr + exponent_e_length]
+  if len(exponent_e) != exponent_e_length: # pragma: no cover
+    raise in_toto.gpg.PacketParsingError("This e MPI has been truncated!")
```

Comment:
This exception message can almost be confused for a typo.  Should it use "exponent e" ?